src/debug/di/rsthread.cpp

```diff
@@ -8325,7 +8325,7 @@ HRESULT CordbJITILFrame::GetNativeVariable(CordbType *type,
 
     HRESULT hr = S_OK;
 
-#if defined(DBG_TARGET_WIN64) || defined(DBG_TARGET_ARM)
+#ifdef WIN64EXCEPTIONS
```

Comment:
Could you please fix the comment on the enclosing `#endif` ?

Reply:
Done!